CHANGELOG.md

```diff
@@ -9,7 +9,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## 6.0.1 - Upgraded `eslint-plugin-backpack`
 
-- Upgrade `eslint-config-backpack` to the latest version.
+- Upgrade `eslint-plugin-backpack` to the latest version.
```

Comment:
Wouldn't it be better to mention the version this is being updated to instead of "latest version"?

Reply:
I think the changes in this commit are self explanatory here 🙂

Reply:
Realised I need to make changes to this change log anyway so will add the versions once I publish 🙂